features/mbedtls/README.md

```diff
@@ -1,14 +1,96 @@
-README for mbed TLS
+README for Mbed TLS
 ===================
 
-mbed TLS for mbed OS
+Mbed TLS for Mbed OS
 --------------------
 
-This edition of mbed TLS has been adapted for mbed OS and imported from its standalone release, which you can find on [github here](https://github.com/ARMmbed/mbedtls). This edition of mbed TLS does not include test code, sample applications, or the scripts used in the development of the library. All of these can be found in the standalone release.
+This edition of Mbed TLS has been adapted for Mbed OS and imported from its standalone release, which you can find on [github here](https://github.com/ARMmbed/mbedtls). This edition of Mbed TLS does not include the test code or the scripts used in the development of the library. All of these can be found in the standalone release.
```

Comment:
“All of these” feels a bit awkward for two things. I think “these” would be better.